### PR TITLE
Fix/anthropic plus models list

### PR DESCRIPTION
**Pull request by JamesHWade**

### CODE REVIEW

**Comment by calderonsamuel** [comment]
Hey James, I don't think we need to export `list_available_models()` methods. I don't even think that we should export the wrapper `get_available_models()` because we don't expect users to use them directly ever. Other than that I think it looks good. Let me know what you think.

**Comment by JamesHWade** [comment]
I added the export becauseI was getting at error with `devtools::document()`. I'll dig in a bit deeper to see what's going on.

**Review by calderonsamuel — APPROVED** [review]
You are right. Maybe we should just merge these changes and then add another issue about "review exported functionality"

**Comment by calderonsamuel** [comment]
This discussion brings more light into the matter: https://github.com/r-lib/roxygen2/issues/1592 

TLDR: The `@export` directive works different with S3 methods compared to regular functions. It is expected that S3 **methods** should always be exported, even if the **generic** is not.

So. We should export the methods but not the generic. Which is what you did, I think

**Comment by JamesHWade** [comment]
Thanks for looking into and for teaching me about the differences in export. I didn't know that.
